src/libostree/ostree-repo-pull.c

```diff
@@ -64,6 +64,7 @@ typedef struct {
   GPtrArray     *meta_mirrorlist;    /* List of base URIs for fetching metadata */
   GPtrArray     *content_mirrorlist; /* List of base URIs for fetching content */
   OstreeRepo   *remote_repo_local;
+  GPtrArray    *repo_refs; /* Array<OstreeRepo> */
```

Comment:
Yeah.  Or maybe skip the git naming and call it `--localcache-repo` in the cmdline and `localcache_repos` in variables?